src/GafferScene/SetAlgo.cpp

```diff
@@ -218,13 +220,41 @@ struct AstEvaluator
 	typedef PathMatcher result_type;
 
 	AstEvaluator( const ScenePlug *scene )
+		: m_scene( scene ), m_wildcardExpression( "\\*" )
 	{
-		m_scene = scene;
 	}
 
 	result_type operator()( const SetName &set ) const
 	{
-		return m_scene->set( set.name )->readable();
+		if( !boost::regex_search( set.name, m_wildcardExpression ) )
```

Comment:
Can we use `StringAlgo::hasWildcards()` here instead? That's what I was expecting, since `StringAlgo` is used for all other matching in Gaffer nodes.